determined-ai/determined
# fix: compare charts showing no data while loading [WEB-1485]

## Description

Charts in the "compare" view should always show a loading state until metrics have been successfully loaded for the selected experiments. 


https://github.com/determined-ai/determined/assets/103522725/ec138e69-fc40-4ffc-a0e2-ee8438b3954e





## Test Plan

1. Navigate to the compare view.
2. With no experiments listed ensure that no charts are showing.
3. Select an experiment with 0 trials, ensure you see "no data to plot"
4. Select an experiment with a trial ensure you see the compare section load, then each individual chart load, and ensure you never see `no data to plot`.
5. De-select all experiments.
6. Click the checkbox in the top left corner of the compare view list and choose `select first 25` ensure that while the charts are loading you do not see `no data to plot` ever. 







## Commentary (optional)





## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket
WEB-1485
